project_parameters.Config.yaml

```
@@ -67,6 +67,7 @@ n_neighbors_value: 20
 variable_value: "ID"
 reference_list_value: NULL
 PCA_Feature_List_value: NULL
+future_globals_value_integration_module: 20971520000 # Seurat: 20971520000, 2000 * 1024^2; Harmony: 1048576000, 1000 * 1024^2; Liger: 1048576000, 1000 * 1024^2
```

Since integration method will be `harmony` for this yaml, should the `future_globals_value_integration_module` value be `1048576000`?

```suggestion
future_globals_value_integration_module: 1048576000 # Seurat: 20971520000, 2000 * 1024^2; Harmony: 1048576000, 1000 * 1024^2; Liger: 1048576000, 1000 * 1024^2
```

---

Thanks for catching this! I edited accordingly, see efb45e6856c7a76d11967cec3809a92630e05b7e.
Please mind that the resource values are subject to change based on the cohort size, so I added a comment about this.